### PR TITLE
workshop-tutorial

### PR DESCRIPTION
**Pull request by ClaireDons**
I made sure that I was able to run the notebooks and adding some additional context, so that they were easier to work with if you are less familiar with xarray etc. 

Also adding some documentation for intro.md and overview.md; including information about cloud data, zarr and the notebooks. Let me know if it is roughly on the right tracks? 

Still to do:
- [x] Change wording for advantages of zarr
- [x] Regridding example is currently really slow
- [x] Information on regridding example

### CODE REVIEW

File: book/intro.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
 (intro)=
+# A tutorial for 'Advancing ecosystem carbon flux research'
+
+This book helps to explore tools and datasets, that are useful for researchers working topics ranging from processes in the carbon cycle to machine learning tools and remote sensing. This book will be tested within the ["Advancing ecosystem carbon flux research"](https://www.lorentzcenter.nl/advancing-ecosystem-carbon-flux-research.html) at the Lorentz Center between 14 - 18 October 2024 and be used to inform the discussion on research priorities for the terrestrial carbon cycle and technical developments for research in the next decade. 
+
+The aim is for the concepts covered during the workshop to open the discussion on openess and reproducibility with the community and to introduce researchers to some useful tools and concepts that can be transfered to their research. 
+
 # Working with modern (cloud) data formats
```
Comment (BSchilperoort) [inline book/intro.md]:
```suggestion
## Working with modern (cloud) data formats
```
Bump this one header level down I think (and then the ones below too)

File: book/intro.md [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+
+Zarr format provides a clean library for accessing the internals of binary files, it also makes those internals themselves simple and transparent. Each n-dimensional chunk of data is given a name and placed in a separate file making it accessible without the need for a library and vastly improving parallelism.
+
+Zarr’s flexible indexing and compatibility with object storage lends itself to parallel processing. 
+
+An added benefit of the simplicity of Zarr is how tractable keeping your data FAIR and accessible in the long-term becomes. Though the bytes need decoding, a fair amount of the general structure needs no interpretation. 
+
+More information on the benefits and uses of Zarr can be found [in this blog](https://medium.com/open-source-science-initiative/why-i-zarr-ee64eb7ffbf8).
+
+Additionally, due to the nature of CSV files, the metadata is included in a separate file and not with the data
```
Comment (BSchilperoort) [inline book/intro.md:23]: Add a reference to the overview.md file here? So people know where to continue

File: book/intro.md [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
+
+The aim is for the concepts covered during the workshop to open the discussion on openess and reproducibility with the community and to introduce researchers to some useful tools and concepts that can be transfered to their research. 
+
 # Working with modern (cloud) data formats
+Geospatial data are becoming increasingly large and complex and as a result, downloading data has become increasingly impractical. Cloud-optimized geospatial formats present an alternative to downloading data, that is more efficient, flexible and scalable. Additionally, it allows the user to more easily download subsets of data, rather than the entire dataset.
+
+An overwiew of the common formats and the advantages of cloud-optimised formats can be found [here](https://guide.cloudnativegeo.org/)
+
+## Zarr
+Currently, many datasets, such as FLUXNET are stored in CSV or netcdf/hdf5 formats, which need to be downloaded by the user from a database requiring an account. This process can be quite time consuming and make it difficult to replicate research if the version, variables used etc are not adequately documented.
```
Comment (BSchilperoort) [inline book/intro.md]: Maybe add some info here on how working with Zarr (as an end user) is quite similar to netCDF

Here's a nice overview of differences: https://help.marine.copernicus.eu/en/articles/8176692-how-to-choose-between-netcdf-and-zarr-format-using-the-toolbox#h_04479906c1

Comment (ClaireDons) [reply book/intro.md]: I maybe addressed this although perhaps it could be more clear.

File: book/intro.md [hunk 4]
```diff
@@ -1,6 +1,26 @@
 (intro)=
-# Working with modern (cloud) data formats
+# A tutorial for 'Advancing ecosystem carbon flux research'
 
-## A tutorial for 'Advancing ecosystem carbon flux research'
+This book helps to explore tools and datasets, that are useful for researchers working topics ranging from processes in the carbon cycle to machine learning tools and remote sensing. This book will be tested within the ["Advancing ecosystem carbon flux research"](https://www.lorentzcenter.nl/advancing-ecosystem-carbon-flux-research.html) at the Lorentz Center between 14 - 18 October 2024 and be used to inform the discussion on research priorities for the terrestrial carbon cycle and technical developments for research in the next decade. 
 
-Empty
+The aim is for the concepts covered during the workshop to open the discussion on openess and reproducibility with the community and to introduce researchers to some useful tools and concepts that can be transfered to their research. A longer overview of the material and sections covered in the workshop can be found [here](https://excited-co2.github.io/workshop_tutorial/main/sections/overview.html)
```
Comment (BSchilperoort) [inline book/intro.md:6]: The link here can be relative. Should be resolved properly when the book is built

**Comment by ClaireDons** [comment]
I ran through the lazy computation notebook and it works well. I really like the example as it makes the regridding very obvious and the diagrams are also useful. I only changed some wording and adding installing the dependencies since it didn't run at first for me. 

I also added a summary for the lazy computation and went through the intro again.

**Comment by BSchilperoort** [comment]
I think the book looks really good now, just have to fix one broken image in the arco era5 notebook:
image

**Pull request merged**